src/components/NavBar.js

```diff
@@ -17,9 +17,9 @@ import { useUsers } from '../utils/users'
 
 // hardcode some demo users
 const identities = {
-  'dfdadc39-7335-404d-af66-c77cf13a15f8': 'Euan',
-  'd64b8476-3c5f-4caf-af6f-9a0f1c51d19f': 'Kris',
-  '2bfaa552-d9a5-41e9-a6c3-5be62b4433c8': 'April'
+  'CiRkZmRhZGMzOS03MzM1LTQwNGQtYWY2Ni1jNzdjZjEzYTE1ZjgSBWxvY2Fs': 'Euan',
```

Comment:
FYI, we're planning to change the keys of the acmecorp/citadel users to be their emails.
If you want to go ahead and make that change now, it will save us the need to update these constants again.

Reply:
Updated